src/main/java/com/lpnu/shaggybeavers/exception/NotExistsUserException.java

```diff
@@ -0,0 +1,7 @@
+package com.lpnu.shaggybeavers.exception;
+
+public class NotExistsUserException extends NotExistsObjectException{
+    public NotExistsUserException() {
+        super("User doesn't exist");
```

Comment:
Add it like constant and constructor withou arg for with that constant message
And construcotr with arg when you check message on blank